drivers/cavium/thunder_mmc.c

```diff
@@ -0,0 +1,425 @@
+/*
+ * Copyright (c) 2016-2017, Cavium Inc. All rights reserved.<BR>
```

Comment:
Please use the standard copyright that the rest of the codebase uses (Arm Limited and Contributors), assuming you are authorised by Cavium to make this contribution on their behalf.

Reply:
No I can't change the license.

Reply:
I was talking the copyright notice, not the license. But you're right that you can't change this if you are not working on behalf of Cavium